dll/cpl/sysdm/lang/zh-HK.rc

```diff
@@ -266,17 +266,18 @@ BEGIN
 END
 
 IDD_ENVIRONMENT_VARIABLES DIALOGEX 6, 18, 252, 245
-STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
+STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUPWINDOW | WS_CAPTION | WS_THICKFRAME | WS_VISIBLE
 CAPTION "環境變數"
 FONT 9, "新細明體", 0, 0, 0x0
 BEGIN
-    GROUPBOX "使用者變數", IDC_STATIC, 7, 12, 238, 100
+    SCROLLBAR IDC_DIALOG_GRIP, 245, 238, 7, 7, SBS_SIZEGRIP
+    GROUPBOX "使用者變數", IDC_USER_VARIABLE_GROUP, 7, 12, 238, 100
     CONTROL "", IDC_USER_VARIABLE_LIST, "SysListView32", LVS_REPORT | LVS_SINGLESEL | LVS_SHOWSELALWAYS |
             LVS_SORTASCENDING | LVS_NOSORTHEADER | WS_VSCROLL | WS_HSCROLL | WS_TABSTOP, 14, 25, 224, 58, WS_EX_CLIENTEDGE
     PUSHBUTTON "新增(&N)", IDC_USER_VARIABLE_NEW, 80, 90, 50, 14
     PUSHBUTTON "編輯(&E)", IDC_USER_VARIABLE_EDIT, 134, 90, 50, 14
     PUSHBUTTON "刪除(&D)", IDC_USER_VARIABLE_DELETE, 188, 90, 50, 14
-    GROUPBOX "系統變數", IDC_STATIC, 7, 116, 238, 100
+    GROUPBOX "系統變數", IDC_SYSTEM_VARIABLE_GROUP, 7, 116, 238, 100
```

Comment:
Heh, it seems this resource part was not updated at the time, thanks for fixing it :)